runtime/hydradx/src/xcm.rs

```diff
@@ -103,28 +104,38 @@ impl Config for XcmConfig {
 
 	type ResponseHandler = PolkadotXcm;
 	type AssetTrap = PolkadotXcm;
+	type AssetLocker = ();
+	type AssetExchanger = ();
 	type AssetClaims = PolkadotXcm;
 	type SubscriptionService = PolkadotXcm;
+	type PalletInstancesInfo = AllPalletsWithSystem;
+	type MaxAssetsIntoHolding = ConstU32<64>;
+	type FeeManager = ();
+	type MessageExporter = ();
+	type UniversalAliases = Nothing;
+	type CallDispatcher = RuntimeCall;
+	type SafeCallFilter = Everything;
```

Comment:
really everything is safe call?

Reply:
Not sure how to set up this one. This is how it looks like in [polkadot](https://github.com/paritytech/polkadot/blob/e05b78689d3fb0deb555ede9f0f0e191cc5e18b5/runtime/polkadot/src/xcm_config.rs#LL184C12-L184C17). Maybe we can start with some simple list of extrinsics (that make sense and we know we want to allow them) and extend it over time.

Reply:
The comment explains how it should be determined:
https://github.com/paritytech/polkadot/blob/e05b78689d3fb0deb555ede9f0f0e191cc5e18b5/runtime/polkadot/src/xcm_config.rs#L177-L183

> Calls that are allowed through this filter must:
> 1. Have a fixed weight;
> 2. Cannot lead to another call being made;
> 3. Have a defined proof size weight, e.g. no unbounded vecs in call parameters.

Reply:
ok, lets solve it after merge in https://github.com/galacticcouncil/HydraDX-node/issues/589